### PR TITLE
docs(readme): add `media-query-splitting-plugin` (Media Query Plugin)

### PR DESCRIPTION
**Pull request by mike-diamond**
Added short note about media-query-splitting-plugin

This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### CODE REVIEW

**Comment by mike-diamond** [comment]
Contributor License Agreement signed

File: README.md [hunk 1]
```diff
@@ -276,6 +276,8 @@ For long term caching use `filename: "[contenthash].css"`. Optionally add `[name
 
 If you'd like to extract the media queries from the extracted CSS (so mobile users don't need to load desktop specific CSS anymore) you should use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin). It will take over the filename option of the mini-css-extract-plugin and recognize its generated CSS chunks.
 
+Or you can use [media-query-splitting-plugin](https://github.com/mike-diamond/media-query-splitting-plugin), it will handle mobile, tablet and desktop styles splitting for each CSS chunk in your bundle without any configuration, just add it to the webpack plugins.
```
Comment (michael-ciniawsky) [inline README.md]: Please make the description more generic e.g

```
- you should use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin) ...
+ you should use one of the following plugins ...
```

and make a list for the plugins instead

```
... its generated CSS chunks.
 
- [Media Query Plugin](media-query-plugin)
- [Media Query Splitting Plugin](media-query-splitting-plugin)

[media-query-plugin]: https://github.com/SassNinja/media-query-plugin
[media-query-splitting-plugin]: https://github.com/mike-diamond/media-query-splitting-plugin
```

File: README.md [hunk 2]
```diff
@@ -274,9 +274,10 @@ For long term caching use `filename: "[contenthash].css"`. Optionally add `[name
 
 ### Media Query Plugin
 
-If you'd like to extract the media queries from the extracted CSS (so mobile users don't need to load desktop specific CSS anymore) you should use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin). It will take over the filename option of the mini-css-extract-plugin and recognize its generated CSS chunks.
+If you'd like to extract the media queries from the extracted CSS (so mobile users don't need to load desktop or tablet specific CSS anymore) you should use one of the the following plugins:
```
Comment (michael-ciniawsky) [inline README.md]:
```diff
- the the following plugins
+ the following plugins
```

File: README.md (hunk 1 above)
Comment (mike-diamond) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (mike-diamond) [reply README.md]: Oh, my bad

**Pull request merged**